changelogs/unreleased/gh-6832-httpc-params-encoding.md

```diff
@@ -0,0 +1,4 @@
+## feature/lua
+
+* Added a new option to pass HTTP query parameters using a Lua table. These
+  parameters are encoded to a query string and passed to HTTP request (gh-6832).
```

Comment:
> pass HTTP query parameters

I guess I can propose a wording variant that is simpler for a reader: 'add query parameters into a URI'. What is 'URI' is known for everyone, but 'query parameter' and names of other URI parts are less widely known.

Feel free to ignore, it is just my thought.

> These parameters are encoded to a query string and passed to HTTP request.

Shouldn't we mention the www-form case, when the parameters are encoded into the request body?